### DISCUSSION TITLE
[Feature] Album made with pictures / video in archive should remain in archive

### DISCUSSION THREAD

**Discussion by Frava735** [immich-app/immich#11546] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When creating an album in archive,  the album itself should be flagged or kept in archives, instead of going with all the other albums.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Albums can't be archived

**Reply by juanpeich** [reply]
There will be a way in the near future to archive an entire album (by doing so, all the photos inside could be archived automatically), so that the person you share the photos with does not see those photos in their Timeline.

It would be ideal until you can choose which albums to share or not to share with your partner.

Thank you very much in advance.
